### PR TITLE
Synchronize the french README with the original content

### PR DESCRIPTION
**Pull request by Mogztter**
Reviews welcomed !

Ping @oncletom 😉

### CODE REVIEW

**Comment by mojavelinux** [comment]
I'm happy to merge. Shall I wait for a review?

**Comment by Mogztter** [comment]
It would be better.
I will do one more round of proofreading 😁

File: README-fr.adoc [hunk 1]
```diff
@@ -58,17 +58,19 @@ endif::[]
...
-Asciidoctor est écrit en Ruby, packagé sous forme de RubyGem et publié sur {uri-rubygem}[RubyGems.org].
-La gemme est aussi incluse dans plusieurs distributions Linux, dont Fedora, Debian, Ubuntu et Alpine.
-Asciidoctor est open source, {uri-repo}[hébergé sur GitHub] et distribué sous {uri-license}[licence MIT].
+{uri-project}/[Asciidoctor] est un processeur de texte et une chaîne de publication _rapide_ et {uri-license}[open source] permettant de convertir du contenu {uri-what-is-asciidoc}[AsciiDoc] en HTML5, DocBook, PDF et d'autres formats.
+Asciidoctor est écrit en Ruby et fonctionne sur les principaux systèmes d'exploitation.
+Pour simplifier l'installation, Asciidoctor est publié au format gem sur {uri-rubygem}[RubyGems.org], et il est également disponible en tant que paquet système sur les principales distributions Linux ainsi que sur macOS.
+Asciidoctor fonctionne aussi sur la JVM avec {uri-asciidoctorj}[AsciidoctorJ] et dans n'importe quel environnement JavaScript avec {uri-asciidoctorjs}[Asciidoctor.js].
```
Comment (Mogztter) [inline README-fr.adoc]: I've added a link on both AsciidoctorJ and Asciidoctor.js
@mojavelinux should we do the same on the English README ?

**Comment by Mogztter** [comment]
I think it's fine, I've added a minor edit but otherwise it looks good 👍

File: README-fr.adoc (hunk 1 above)
Comment (mojavelinux) [reply README-fr.adoc]: Yep, I'll make that update. Thanks for the suggestion.

**Comment by mojavelinux** [comment]
Nice work! Let's merge it.

**Comment by oncletom** [comment]
I'm late to the party but I'm happy to give it a read this Friday if you want.

**Comment by Mogztter** [comment]
Sure that would be great, thanks Thomas

**Comment by mojavelinux** [comment]
You're never too late in open source ;)

**Pull request merged**